api/v1/installation_types.go

```diff
@@ -162,8 +162,21 @@ type InstallationSpec struct {
 
 	// CalicoWindowsUpgradeDaemonSet configures the calico-windows-upgrade DaemonSet.
 	CalicoWindowsUpgradeDaemonSet *CalicoWindowsUpgradeDaemonSet `json:"calicoWindowsUpgradeDaemonSet,omitempty"`
+
+	// FIPSMode uses images and features only that are using FIPS 140-2 validated cryptographic modules and standards.
+	// Default: Disabled
+	// +kubebuilder:validation:Enum=Enabled;Disabled
+	// +optional
+	FIPSMode FIPSMode `json:"fipsMode,omitempty"`
```

Comment:
I think this should be a pointer, right?

Reply:
I think you are right. Let me change that.

I just did a quick evaluation on the string aliases that we have for installation_types. We are not very consistent currently.

We don't use pointers for:
- ProductVariant
- Provider
- ComponentName
- EncapsulationType
- NATOutgoingType
- CNIPluginType
- IPAMPluginType

But we do for:
- NonPrivilegedType
- ContainerIPForwardingType
- HostPortsType
- MultiInterfaceMode
- BGPOption
- LinuxDataplaneOption
- KubernetesAutodetectionMethod

Reply:
The resulting CRD is identical.